contributing/projet-management.md

```diff
@@ -0,0 +1,105 @@
+| 4. | “Implementation Brief” is added to the issue description, and the issue is moved to the “Implementation Brief Review” column. NB: an estimate (step 6) may be added as part of this step. | `Engineer`
+| 5. | The “Implementation Brief” is reviewed and the issue is moved to the “Estimate” column upon approval. The issue is moved back to the “Implementation Brief” column if changes in the “Implementation Brief” description are requested. | `Lead Engineer`
+| 6. | The issue is estimated using T-Shirt sizing and moved to the [Execution](https://github.com/ampproject/amp-wp/projects/15) project board (automatically added to the Backlog). | `Project Manager` `Engineer`
+
+##### Execution
+| <br> Step | <br> Task | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br> Role  |
+| :--- | :--- | :--- |
+| 1. | An issue requiring work is added to the [Execution](https://github.com/ampproject/amp-wp/projects/15) project board (automatically added to the Backlog) after going through the [Definition](https://github.com/ampproject/amp-wp/projects/14) project board pipeline. | `Product Manager` `Program Manager`
+| 2. | The issue is assigned a “[Milestone](https://github.com/ampproject/amp-wp/milestones)” and “Sprint” label. | `Product Manager` `Program Manager`
+| 3. | The issue is moved to the “To Do” column if it is assigned to the current sprint. | `Project Manager`
+| 4. | The issue is assigned (or may be self-assigned) to an engineer. | `Project Manager` `Engineer`
+| 5. | The issue is moved to the “In Progress” column when development starts. A PR is created, following the [Branching Strategy](https://github.com/ampproject/amp-wp/contributing/engineering.md#branches). The PR must contain details for each section predefined in the PR template, with a reference to the associated issue. **IMPORTANT:** do not add [GitHub keywords](https://help.github.com/en/articles/closing-issues-using-keywords) which would automatically close an issue once the PR is merged. | `Engineer`
+| 6. | The “[Changelog Message](https://github.com/ampproject/amp-wp/contributing/engineering.md#changelog)” is added to the issue description and the issue is moved to the “Code Review” column once development is completed. | `Engineer`
+| 7. | The code review is done in the referred PR and the issue is moved to the “QA” column once the review is completed and the PR is approved, merged and deployed to the QA environment. The reviewer must ensure that the “Acceptance Criteria” match the implementation before moving the issue to QA. | `Engineer`
+| 8. | The issue is moved to the “Approval” column once QA is passed or moved back to the “To Do” column if changes are required, in which case the cycle from the “To Do” column onwards is repeated. | `QA Specialist`
```

Comment:
This is the step when we would add a demo screencast - do you want to add that into the documentation here?